x-pack/solutions/observability/plugins/apm/server/routes/apm_routes/register_apm_server_routes.ts

```diff
@@ -229,7 +230,7 @@ export function registerRoutes({
           path: pathname,
           options,
           validate: passThroughValidationObject,
-          security,
+          security: security as RouteSecurity,
```

Comment:
As of its interface `security` is already a type of `RouteSecurity`, but it is marked as optional. Should we update the interface instead if we are sure it'll never be undefined?

<img width="666" alt="Screenshot 2025-03-21 at 08 50 43" src="https://github.com/user-attachments/assets/6f592d59-6550-459e-9044-c82d1cf58395" />

Reply:
Ah, just seen @miltonhultgren 's comment mentioning he opened an issue to fix the interface. https://github.com/elastic/observability-dev/issues/4415